tests/test_auth_api.py

```
@@ -0,0 +1,8 @@
+from jose import JWTError
+from fastapi import HTTPException
+
+try:
+    # Simulons un problème avec le token
+    raise JWTError("invalid signature")
+except  HTTPException:
```

Flake8 (7.2.0)

[error] multiple spaces after keyword

(E271)